startup/GafferScene/cyclesLights.py

```diff
@@ -61,14 +61,19 @@
 Gaffer.Metadata.registerValue( "cycles:light:quad_light", "exposureParameter", "exposure" )
 Gaffer.Metadata.registerValue( "cycles:light:quad_light", "colorParameter", "color" )
 Gaffer.Metadata.registerValue( "cycles:light:quad_light", "textureNameParameter", "image" )
+Gaffer.Metadata.registerValue( "cycles:light:quad_light", "widthParameter", "sizeu" )
+Gaffer.Metadata.registerValue( "cycles:light:quad_light", "heightParameter", "sizev" )
 
 Gaffer.Metadata.registerValue( "cycles:light:portal", "type", "quad" )
+Gaffer.Metadata.registerValue( "cycles:light:portal", "widthParameter", "sizeu" )
+Gaffer.Metadata.registerValue( "cycles:light:portal", "heightParameter", "sizev" )
 
 Gaffer.Metadata.registerValue( "cycles:light:disk_light", "type", "disk" )
 Gaffer.Metadata.registerValue( "cycles:light:disk_light", "intensityParameter", "intensity" )
 Gaffer.Metadata.registerValue( "cycles:light:disk_light", "exposureParameter", "exposure" )
 Gaffer.Metadata.registerValue( "cycles:light:disk_light", "colorParameter", "color" )
-Gaffer.Metadata.registerValue( "cycles:light:disk_light", "widthParameter", "size" )
+Gaffer.Metadata.registerValue( "cycles:light:disk_light", "widthParameter", "sizeu" )
+Gaffer.Metadata.registerValue( "cycles:light:quad_light", "heightParameter", "sizev" )
```

Comment:
There's duplication here of the quad_light `heightParameter` registration, though I think you'll find that if you change this to `cycles:light:disk_light` it'll have no effect as the disk light [visualiser](https://github.com/GafferHQ/gaffer/blob/c618bef83170b5c5a5221cc03f95870ba03a99cb/src/GafferSceneUI/StandardLightVisualiser.cpp#L562-L563) only supports `widthParameter` or `radiusParameter`. 

Do you think it's worth keeping the `disk_light` limited to width for now rather than presenting a potentially confusing visualisation? UsdLuxDiskLight also only supports radius, so I don't think we'd be losing anything when it comes to any potential future USD conversions that may be cooking. :)

Reply:
Oops copy-pasta typo. Yep I've got it set now to just apply width to both sizeu & sizev now.